app/pages/lab/visibility.cjsx

```diff
@@ -17,11 +16,18 @@ module.exports = React.createClass
       private: false
       beta_requested: false
       launch_requested: false
+    workflows: null
```

Comment:
I refactored `workflows` to not use the `SetToggle` mixin since I added the custom handling to remove the project default config if necessary. I added some curly braces to make it clear what is what here.